java/runtime/src/main/java/io/ray/runtime/object/OwnershipInfo.java

```diff
@@ -0,0 +1,46 @@
+package io.ray.runtime.object;
+
+import java.io.Externalizable;
+import java.io.IOException;
+import java.io.ObjectInput;
+import java.io.ObjectOutput;
+
+/// A POJO that's used to inline some serialized information for object ownership.
+public final class OwnershipInfo implements Externalizable {
```

Comment:
Sorry that I misled you. I meant `Serializable`, not `Externalizable`. I don't think we need to serialize fields manually.